frontend/src/components/Post/Post.js

```diff
@@ -10,6 +10,7 @@ import PostImage from "./PostImage";
 import timeElapsedSincePosted from "../../services/TimeService";
 
 import "./Post.scss";
+import LikePicture from '../LikeButtonPicture/LikeButtonPicture';
```

Comment:
Can you put this with the rest of the project imports and sort it alpahebtically? (between FollowingButton and PostComment, really)